dask_cloudprovider/providers/aws/ecs.py

```diff
@@ -656,12 +680,18 @@ async def _start(self,):
         if self._scheduler_timeout is None:
             self._scheduler_timeout = self.config.get("scheduler_timeout")
 
+        if self._scheduler_extra_args is None:
+            self._scheduler_extra_args = self.config.get("scheduler_extra_args")
```

Comment:
Can you please add the two extra config options to the example config file, commented out and with a brief description

Reply:
Done. Also, added splitting of comma-separated arguments if loaded from config file. (Arguments in existing code are in a list and I extended that list with any extra arguments.)